### DISCUSSION TITLE
What is the font family used in the icons preview image?

### DISCUSSION THREAD

**Discussion by oliveiralecca** [prazdevs/catppuccin-vscode-icons#356] [Q&A] [answered]
<img width="730" alt="Screenshot 2024-11-09 at 01 18 09" src="https://github.com/user-attachments/assets/2089eb41-26b4-4dcf-b36d-e4fe3456ed35">

**Comment by doas-ice** [comment, marked as answer]
Sorry for necro-bumping this old post.

I was also looking for this but found nothing.

After inspecting the preview generation workflow and preview generation [script](https://github.com/catppuccin/vscode-icons/blob/chore/re-generate-previews/scripts/icons/preview.ts) it seems the font used is called [Space Mono](https://fonts.google.com/specimen/Space+Mono).

Hope this can help someone find it in the future.

**Reply by oliveiralecca** [reply]
Oh my goodness! I'm really grateful for that!!
